source/install/desktop-msi-gpo.rst

```diff
@@ -120,3 +120,12 @@ Verify Group Policy settings in the installed Desktop app
 2. Verify the app loads the first server defined in the ``Edit group policy`` app.
 
    .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00018.png
+
+Silent Installation Guide
+--------------------------------------------
+
+Silent installation of MSI can be performed with the following command:
+
+``msiexec /i mattermost-desktop-v4.6.1-x64.msi /qn``
+
+Note: Current version is 4.6.1. In future, you may need to change it accordingly.
```

Comment:
Very minor update: "In **the** future, you may need to change it accordingly."

Reply:
Done! Thank you:)